### PR TITLE
Add title sort to catalog view (by last name for people)

### PR DESCRIPTION
**Pull request by 13miciunasl**
Closes uclibs/scholar_uc#573

### CODE REVIEW

File: app/repository_models/person.rb [hunk 1]
```diff
@@ -123,6 +123,7 @@ def user
   def to_solr(solr_doc={}, opts={})
     super(solr_doc, opts)
     Solrizer.set_field(solr_doc, 'generic_type', 'Person', :facetable)
+    Solrizer.set_field(solr_doc, 'sort_title', "#{self.last_name}, #{self.first_name}", :stored_sortable)
```
Comment (jamesvanmil) [inline app/repository_models/person.rb]: There's a method that gives the inverted name. It's technically on the user model, so you'd have to hop to that, but it handles cases where the name may not be set, or where a field is blank:

https://github.com/uclibs/curate/blob/develop/app/models/curate/user_behavior/base.rb#L54-L58

**Comment by jamesvanmil** [comment]
It doesn't look like this handles leading articles for work or collection title (e.g., "The...", "A...", "An..."). We should be covering these in English, at least.

Also - I'm thinking about specs... I know that there weren't any in place for the existing sorts. But that doesn't mean that there _shouldn't_ be any here. Let's plan to talk about this in our next stand-up.

**Comment by jamesvanmil** [comment]
@hortongn you're going to give feedback about need for specs

**Comment by hortongn** [comment]
I'm fine not testing the sort itself, but can we get a test that verifies "title" is showing up as one of the options in the sort dropdown menu?  Something like `expect(page).to have_select("sort", with_options: ['title'])`

**Pull request merged**